sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_first_seen_v1/init.sql

```diff
@@ -1,144 +1,24 @@
-CREATE OR REPLACE TABLE
-  fxa_users_services_first_seen_v1
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.firefox_accounts_derived.fxa_users_services_first_seen_v1`
 PARTITION BY
-  DATE(first_service_timestamp)
+  first_seen_date
 CLUSTER BY
   service,
-  user_id
+  os_name,
+  country
 AS
-WITH base AS (
-  SELECT
-    * REPLACE (
-      IF(service IS NULL AND event_type = 'fxa_activity - cert_signed', 'sync', service) AS service
-    )
-  FROM
-    firefox_accounts.fxa_content_auth_oauth_events
-),
-  -- use a window function to look within each USER and SERVICE for the first value of service, os, and country.
-  -- also, get the first value of flow_id for later use and create a boolean column that is true if the first instance of a service usage includes a registration.
-  -- [kimmy] the variable first_service_timestamp_last is named so because it is actually the last timestamp recorder in the user's first flow,
-  -- NOT the first timestamp in their first flow.
-  -- it's used later on to order by service, so i'm keeping it here and just renaming it.
-first_services AS (
-  SELECT
-    ROW_NUMBER() OVER w1_unframed AS _n,
-    user_id,
-    service,
-      -- using mode_last with w1_reversed to get mode_first
-    udf.mode_last(ARRAY_AGG(`timestamp`) OVER w1_reversed) AS first_service_timestamp_last,
-    udf.mode_last(ARRAY_AGG(flow_id) OVER w1_reversed) AS first_service_flow,
-    LOGICAL_OR(IFNULL(event_type = 'fxa_reg - complete', FALSE)) OVER w1_reversed AS did_register
-  FROM
-    base
-  WHERE
-    (
-      (event_type IN ('fxa_login - complete', 'fxa_reg - complete') AND service IS NOT NULL)
-      OR (event_type LIKE 'fxa_activity%')
-    )
-    AND DATE(`timestamp`) >= '2019-03-01'
-    AND user_id IS NOT NULL
-  WINDOW
-    -- We must provide a window with `ORDER BY timestamp DESC` so that udf.mode_last actually aggregates mode first.
-    w1_reversed AS (
-      PARTITION BY
-        user_id,
-        service
-      ORDER BY
-        `timestamp` DESC
-      ROWS BETWEEN
-        UNBOUNDED PRECEDING
-        AND UNBOUNDED FOLLOWING
-    ),
-    -- We must provide a modified window for ROW_NUMBER which cannot accept a frame clause.
-    w1_unframed AS (
-      PARTITION BY
-        user_id,
-        service
-      ORDER BY
-        `timestamp`
-    )
-),
-  -- we need this next section because `did_register` will be BOTH true and false within the flows that the user registered on.
-  -- this dedupes the rows from above and sets did_register to true only on flows that included a registration
-  -- I've verified that `date(first_service_timestamp), count(distinct user_id) where did_register = true group by 1`  matches the counts of registrations per day in amplitude.
-first_services_g AS (
-  SELECT
-    * EXCEPT (_n)
-  FROM
-    first_services
-  WHERE
-    _n = 1
-),
-  -- sadly, `entrypoint` is null on registration complete and login complete events.
```

Comment:
@kik-kik just looking through the parts we are not keeping. Given this comment, does this mean that the entrypoint field in the table we currently have will always be null?